internal/api/cluster.go

```diff
@@ -0,0 +1,150 @@
+package api
```

Comment:
I imagine this file doesn't register it's own handlers to avoid having the router globally available in the package?

Reply:
At the time, I had left everything in api.go, thinking it would actually be easier to introspect the endpoints if they were in one place instead of scattered across multiple files. 0/5, but maybe we defer until the "next" API endpoint and see how that looks?

Reply:
Good with me 👍